pkg/deb/usr/lib/systemd/system/peerbanhelper.service

```diff
@@ -5,7 +5,7 @@ After=network.target
 [Service]
 User=peerbanhelper
 WorkingDirectory=/etc/peerbanhelper
-ExecStart=/usr/bin/java -Dpbh.release=debian -Dpbh.datadir=/etc/peerbanhelper -Dpbh.log.level=WARN -Xmx512M -XX:+UseG1GC -XX:+UseStringDeduplication -XX:+ShrinkHeapInSteps -jar /usr/lib/peerbanhelper/PeerBanHelper.jar
+ExecStart=/usr/bin/java -Dpbh.release=debian -Dpbh.datadir=/opt/peerbanhelper -Dpbh.configdir=/etc/peerbanhelper -Dpbh.logdir=/var/log/peerbanhelper -Dpbh.log.level=WARN -Xmx512M -XX:+UseG1GC -XX:+UseStringDeduplication -XX:+ShrinkHeapInSteps -jar /usr/lib/peerbanhelper/PeerBanHelper.jar
```

Comment:
the `datadir` should be `/var/lib/peerbanhelper`, don't forget to create those dir~

Reply:
I went with `/opt` instead but if you like `/var/lib` more I can change it to that.

Reply:
It's in `/var/lib` now.